src/corona_hakab_model/supervisor.py

```diff
@@ -130,7 +168,7 @@ def scale(self) -> Optional[Tuple[float, float]]:
         pass
 
     @abstractmethod
-    def plot(self, ax):
+    def plot(self, ax, prefix="", style=""):
```

Comment:
I'm not sure about this, Maybe it's better to override name and style with a special class and not allow the supervisor free reign. We should talk about this change